src/page_body/support/page_body_scripts.pr

```diff
@@ -51,7 +51,7 @@ $(document).ready(function() {
     const cookieConsent = new CookieConsent({
         content: {
             "title": "{{ escapeHtml(configuration.cookiesBannerTitle) }}",
-            "body": "{{ addSlashes(configuration.cookiesBannerDescription) }}",
+            "body": "{{ withHTMLNewlines(escapeHtml(addSlashes(configuration.cookiesBannerDescription))) }}",
```

Comment:
This will not work, because the cookie banner needs to support HTML (check images with what this code causes). That's why there was `addSlashes` and not `escapeHtml` originally.

![CleanShot 2023-11-01 at 09 33 00](https://github.com/Supernova-Studio/exporter-documentation/assets/6851247/2c360311-594a-4959-a635-ed37f5801dd2)

Reply:
I fixed it by removing `escapeHtml`. Was there a specific reason why you added it?

Reply:
No, just so this function and thought that it might be useful